packages/terra-table/src/subcomponents/Cell.module.scss

```diff
@@ -0,0 +1,42 @@
+/* stylelint-disable terra/custom-property-namespace */
```

Comment:
You should not be disabling linter rules.  You just need to update the names to resolve appropriately.